examples/Examples.idr

```diff
@@ -5,6 +5,7 @@ import Types
 import Typedefs
 import TermCodec
 import Backend.Haskell
+import Backend.ReasonML
 
 ------ example: bits -
```

Comment:
Why does this comment start with 5 dashes followed by a lowercase letter, as opposed to the others, which look like `-- Example:`?

Reply:
I was wondering the same thing when I was editing the file, but figured that you had your reasons. Can absolutely edit if desired!